docs/architecture/ADRs/tile_server.md

```diff
@@ -0,0 +1,30 @@
+# Tile server
+
+## Status
+In progress
+
+## Context
+The main feature of the Vector Atlas webpage will be an interactive map of Africa, allowing users to interact visually with the spatial data. The choice of the map library and underlying tile server are key to the success of the project. This ADR discusses the choice of tile server.
+
+### Options
```

Comment:
These choices need to be expanded slightly - while the maps for some of them are free, there are usage restrictions on the apis that provide the tiles. We should document these limits, particularly in the OpenStreetMap case.

Reply:
This PR is not for this issue - I'll open another PR for those changes.

Reply:
ah ok, no that's fine if we're still going to expand this further else where.